nucypher-core-python/nucypher_core/umbral.pyi

```diff
@@ -42,10 +40,11 @@ class SecretKeyFactory:
         ...
 
     @staticmethod
-    def from_secure_randomness(data: bytes) -> SecretKeyFactory:
+    def from_secure_randomness(seed: bytes) -> SecretKeyFactory:
```

Comment:
I see you decided to use a different name here compared to the ferveo key above - `seed` is a better name for this function considering it's intended usage.